task/tekton-operator-install/0.1/tekton-operator-install.yaml

```diff
@@ -0,0 +1,187 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: tekton-operator-install
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: operator, tekton, deploy
+    tekton.dev/displayName: "Tekton Operator Install"
+spec:
+  description: >-
+    This task can be used to install Tekton pipelines and also it's
+    components using Tekton Operator on a new cluster.
+  workspaces:
+    - name: kubeconfig
+      description: >-
+        The workspace consisting of kubeconfig file so that operator
+        can install tekton components on fresh cluster
+  params:
+    - name: OPERATOR_VERSION
+      type: string
+      description: Tekton operator version to be installed
+      default: "latest"
+    - name: INSTALL_TRIGGERS
+      type: string
+      description: The version of the triggers to be installed
+      default: "false"
+    - name: INSTALL_DASHBOARD
+      type: string
+      description: The version of the Tekton dashboard to be installed
+      default: "false"
+    - name: INSTALL_EXTENSION_WEBHOOKS
+      type: string
+      description: >-
+        The version of the Tekton Extension Webhooks to be installed
```

Comment:
:+1: for switching to boolean params
could you update the description(s) as well

Reply:
addressed :+1: